src/extract.ts

```diff
@@ -134,11 +134,9 @@ export interface PytestBenchmarkJson {
 }
 
 export interface CustomBenchmarkJson {
```

Comment:
Thanks for this! I was actually trying to follow the pattern of `GoogleCppBenchmarkJson` and `PytestBenchmarkJson` which _do_ have `benchmarks` as top level keys. But your change inspired me to read the rest of the file and I think it reads very well to just reuse the `BenchmarkJson` class type like I mentioned in my previous comment!

What do you think?

Reply:
> Thanks for this! I was actually trying to follow the pattern of GoogleCppBenchmarkJson and PytestBenchmarkJson which do have benchmarks as top level keys

Ah, that makes sense. I didn't explore the other interface enough, sorry. 

> This means we can delete this type entirely

Super catch, thank you! 

> Although I don't know what we would use the range and extra keywords for 

Same. Using the `BenchmarkResult` type actually made things easier since they were already marked as optional types. Now we can ignore setting them safely 🙊